nevergrad/benchmark/plotting.py

```diff
@@ -638,6 +667,16 @@ def winrates_from_selection(
         num_rows = min(num_rows, len(all_optimizers))
         # iterate on all sub cases
         victories, total = aggregate_winners(df, categories, all_optimizers)
+        if restricted_to_complete:
+            print([int(2 * victories.loc[n, n]) for n in all_optimizers])
+            max_num = max([int(2 * victories.loc[n, n]) for n in all_optimizers])
+            new_all_optimizers = [n for n in all_optimizers if int(2 * victories.loc[n, n]) == max_num]
+            print(new_all_optimizers)
+            print("vs")
+            print(all_optimizers)
```

Comment:
please clean the prints and add a dosctring to explain what "restricted_to_complete" does. Maybe "only_completed" is simpler and clearer?

Reply:
Done.